apis/tempo/v1alpha1/tempomonolithic_types.go

```diff
@@ -277,14 +282,36 @@ type MonolithicJaegerUIRouteSpec struct {
 	Host string `json:"host,omitempty"`
 
 	// Termination specifies the termination type.
-	// Default: edge.
 	//
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=edge
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="TLS Termination"
 	Termination TLSRouteTerminationType `json:"termination,omitempty"`
 }
 
+// MonolithicMultitenancySpec defines the multi-tenancy settings for Tempo.
+type MonolithicMultitenancySpec struct {
+	// Enabled defines if multi-tenancy is enabled.
+	//
+	// +kubebuilder:validation:Required
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Enabled",xDescriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
+	Enabled bool `json:"enabled"`
+
+	TenantsSpec `json:",inline"`
```

Comment:
missing docs?

Reply:
It's an inline field, docs are at the properties of the inlined struct